terraform/modules/fhir/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "fhir_dataset_id" {
+  value = google_healthcare_dataset.dataset.name
+}
+
+output "fhir_store_id" {
+  value = google_healthcare_fhir_store.default.name
```

Comment:
Same here, should be `.id`: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/healthcare_fhir_store#attributes-reference

Reply:
Actually, if you look at what we are after name works. ID will contain the full project_id/dataset_id/ etc...

id - an identifier for the resource with format projects/{{project}}/locations/{{location}}/datasets/{{name}}

vs

name - (Required) The resource name for the Dataset.